src/components/hint/index.tsx

```diff
@@ -67,7 +67,7 @@ export interface HintProps {
   /**
    * The hint message
    */
-  message?: string | ReactElement;
+  message?: string | ReactElement | (string | ReactElement | null)[];
```

Comment:
```suggestion
  message?: string | ReactElement | (string | ReactElement)[];
```
Hi Andrew, is there a reason you are using `null` as type ?

Reply:
I'd use ReactNode instead of all of those, TBH. Includes string, ReactElement and null. Less restrictive
```suggestion
  message?: ReactNode | ReactNode[];
```

Reply:
@adids1221 it allows optional parts of the string like `['No items', null]` or `['No items', <Text onPress={}>Add</Text>]`

Reply:
I agree with @ipanasenko.
@andykog change it to `ReactNode `

Reply:
@adids1221 updated, note that now it will allow passing `message={<View>...</View>}` which is not valid

Reply:
It was already allowed before by ReactElement type, no?

Reply:
It was indeed, probably the idea was to allow passing fragments